node/rpc/src/state_api.rs

```diff
@@ -304,7 +286,7 @@ pub(crate) async fn state_account_key<
     let state = state_for_ts(&state_manager, Some(tipset))?;
     let address =
         interpreter::resolve_to_key_addr(&state, state_manager.get_block_store_ref(), &actor)?;
-    Ok(address.into())
+    Ok(Some(address))
```

Comment:
I can't comment on previous, but won't this be incorrect serialization because it's going to serialize as bytes? Default json serialization should be string?